Exonoesis/Leviathan-Logic-Engine
# Convert BackgroundTesting.cs into PlayMode Test Method

To test this change, open the TestRunner by:
1. Selecting `Window` in the menu bar
2. Under General, select `Test Runner`
3. Select the `Play Mode` tab
4. Run `testBackgroundViewerTransition` by double clicking.
5. Observe void background turning into a forest one.
6. Witness a Green check mark to the left of the test entry on the Test Runner.

Changes:
- Added testBackgroundViewerTransition in PlayTestSuite.cs
- Added tag to EventSystem for ^.
- Remove BackgroundTesting.cs
- Updated Unity version for project.
